### PR TITLE
Add a note about advanced mode

### PR DESCRIPTION
**Pull request by Gaff**
It took me about 2 hours this morning to figure out where this addon had gone to. Let's give users a sporting chance to figure it out!

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @Gaff, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;hassio-addons#1255).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: ssh/README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ ha help
 
 Follow these steps to get the add-on installed on your system:
 
+0. This addon is only enabled for "Advanced Mode" users. To enable: **Profile** -> **Advanced Mode**
```
Comment (frenck) [inline ssh/README.md:22]: Please re-number the other items on the list as well.

```suggestion
`1. This addon is only visible to "Advanced Mode" users. To enable advanced mode, go to **Profile** -> and turn on **Advanced Mode**.
```

Comment (frenck) [reply ssh/README.md]: You've only partly applied my suggestion. Any specific reason?

Comment (Gaff) [reply ssh/README.md]: Sorry wasn't paying attention 😖

**Pull request merged**